### DISCUSSION TITLE
Cannot add the module (recently installed).

### DISCUSSION THREAD

**Discussion by JoseFMNobre** [openstore-ecommerce/OpenStore#275] [Q&A] [answered]
Greetings,

I just installed OpenStore on our new website but cannot add any of the modules and keep receiving the following error: "DotNetNuke.Data.SqlDataProvider, DotNetNuke".
![os error](https://user-images.githubusercontent.com/87533076/178553271-3a93e8dd-cd9b-4053-8e5b-604b901c6d39.PNG)

I greatly appreciate any advice.

Best regards,
José Nobre

**Comment by JoseFMNobre** [comment, marked as answer]
I've sent an email to the Technical contact and was able to resolve the issue.

**Comment by DNNMonster** [comment]
Good news.  Thanks for reporting back.  What was the root cause?

**Reply by JoseFMNobre** [reply]
I did not follow the guide and didn't install "NBright Template System"
